api/src/main/java/org/infinispan/api/async/AsyncContainer.java

```diff
@@ -0,0 +1,25 @@
+package org.infinispan.api.async;
+
+import java.util.concurrent.Flow;
+
+import org.infinispan.api.Infinispan;
+import org.infinispan.api.common.events.container.ContainerEvent;
+import org.infinispan.api.common.events.container.ContainerListenerEventType;
+
+/**
+ * @since 14.0
+ **/
+public interface AsyncContainer extends Infinispan {
+
+   AsyncCaches caches();
+
+   AsyncMultiMap multiMaps();
```

Comment:
`<K, V> AsyncMultiMap<K, V> multiMaps();` So we can do `AsyncMultiMap<String, String> multiMap = c.multiMaps();`

Reply:
Done